src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -936,10 +936,11 @@ def test_webapp_list_locations_free_sku(self, resource_group):
 
 class WebappTriggeredWebJobListTest(ScenarioTest):
     @ResourceGroupPreparer()
+    @record_only()
     def test_webapp_triggeredWebjob_list(self, resource_group):
         # testing this using a webjob already created
         # given there is no create command inorder to re-record please create a webjob before
-        # recording this
+        # recording this. Once the create command is available, please remove the "record_only" flag
```

Comment:
We are not planning on supporting a create command for webjobs at this time. so may be remove this comment?

Reply:
We never know :), maybe external users would contribute it...